modules/apps/static/portal-file-install/portal-file-install-test/src/testIntegration/java/com/liferay/portal/file/install/deploy/test/FileInstallConfigTest.java

```diff
@@ -307,6 +307,46 @@ public void testFactoryConfigurationWithUnderline() throws Exception {
 		_testFactoryConfiguration(CharPool.UNDERLINE);
 	}
 
+	@Test
+	public void testMultipleFactoryConfigurations() throws Exception {
+		_testFactoryConfiguration(CharPool.TILDE);
```

Comment:
Re-read through this function to make sure you know what it does. It technically created a "hard-coded" key/value pair for a factory config, with some seperator being used (looks like class name? didnt look too much into that).